easybuild/easyconfigs/h/Hello/Hello-2.10-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'Hello'
+version = '2.10'
+
+homepage = 'https://www.gnu.org/software/hello/'
+
+description = """
+The GNU Hello program produces a familiar, friendly greeting. Yes, this is another
+implementation of the classic program that prints "Hello, world!" when you run it.
+
+However, unlike the minimal version often seen, GNU Hello processes its argument
+list to modify its behavior, supports greetings in many languages, and so on.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['31e066137a962676e89f69d1b65382de95a7ef7d914b8cb956f41ea72e0f516b']
+
+# use same binutils version that was used when building GCCcore toolchain
+builddependencies = [
+    ('binutils', '2.31.1'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/hello'],
+    'dirs': ['share/man/man1'],
+}
+sanity_check_commands = ['hello']
+
+moduleclass = 'system'
```

Comment:
@bartoldeman `tools` makes more sense to me, indeed (certainly more than `system`)

I do wonder what your use case was here? :)